docs/en/integrations/data-ingestion/kafka/cloud/confluent/kafka-connect-http.md

```diff
@@ -0,0 +1,139 @@
+    assignee LowCardinality(String),
+    assignees Array(LowCardinality(String)),
+    closed_at DateTime,
+    merged_at DateTime,
+    merge_commit_sha String,
+    requested_reviewers Array(LowCardinality(String)),
+    merged_by LowCardinality(String),
+    review_comments UInt32,
+    member_login LowCardinality(String)
+) ENGINE = MergeTree ORDER BY (event_type, repo_name, created_at)
+
+```
+
+#### 4. Add data to Kafka
+
+Insert messages to Kafka. Below we use [kcat](https://github.com/edenhill/kcat) to insert 10k messages.
+
+```bash
+head -n 10000 github_all_columns.ndjson | kafkacat -b <host>:<port> -X security.protocol=sasl_ssl -X sasl.mechanisms=PLAIN -X sasl.username=<username>  -X sasl.password=<password> -t github
+```
+
+A simple read on the target table “Github” should confirm the insertion of data.
+
+
+```sql
+SELECT count() FROM default.github;
+
+| count\(\) |
+| :--- |
+| 10000 |
+
+```
+
+## Confluent Cloud
+
+A fully managed version of HTTP Sink is available for those using Confluent Cloud for their Kafka hosting. This requires your ClickHouse environment to be accessible from Confluent Cloud. We assume you have taken the appropriate measures to secure this.
+
+The instructions for creating an HTTP Sink in Confluent Cloud can be found [here](https://docs.confluent.io/cloud/current/connectors/cc-http-sink.html). The following settings are relevant if connecting to ClickHouse. If not specified, form defaults are applicable:
+
+
+* `Input messages` - Depends on your source data but in most cases JSON or Avro. We assume JSON in the following settings.
+* `Kafka Cluster credentials` - Confluent cloud allows you to generate these for the appropriate topic from which you wish to pull messages.
+* HTTP server details - The connection details for ClickHouse. Specifically:
+    * `HTTP Url` - This should be of the same format as the self-managed configuration parameter `http.api.url i.e. &lt;protocol>://&lt;clickhouse_host>:&lt;clickhouse_port>?query=INSERT%20INTO%20&lt;database>.&lt;table>%20FORMAT%20JSONEachRow`
+    * `HTTP Request Method` - Set to POST
+    * `HTTP Headers` - “Content Type: application/json”
+* HTTP server batches
+    * `Request Body Format` - json
+    * `Batch batch size` - Per ClickHouse recommendations, set this to at least 1000.
+* HTTP server authentication
+    * `Endpoint Authentication type` - BASIC
+    * `Auth username` - ClickHouse username
+    * `Auth password` - ClickHouse password
+* HTTP server retries - Settings here can be adjusted according to requirements. Timeouts specifically may need adjusting depending on latency.
+    * `Retry on HTTP codes` - 400-500 but adapt as required e.g. this may change if you have an HTTP proxy in front of ClickHouse.
+    * `Maximum Reties` - the default (10) is appropriate but feel to adjust for more robust retries.
+
+<img src={require('./images/http_sink_config.png').default} class="image" alt="Connecting Confluent HTTP Sink" style={{width: '50%'}}/>
```

Comment:
The file was removed in https://github.com/ClickHouse/clickhouse-docs/pull/770/ 3 weeks ago

Reply:
I figured it out. I accidentally re-added these Kafka files (bad copy-and-pasting). Thanks for noticing!